frontend/src/pages/home.pages.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+
+import { RouteComponentProps } from "@reach/router";
+
+type TParams = { id: string };
+
+export const HomePage: FC<RouteComponentProps> = (props: RouteComponentProps<TParams>) => {
+  console.log(props)
```

Comment:
There is a console log here. But most importantly, let's keep the Dashboard page only for now.

Reply:
It looks like the home page is still here.